.readme-partials.yaml

```diff
@@ -134,3 +134,50 @@ custom_content: |
   ```
   com.google.cloud.examples.logging.snippets.AddLoggingHandler.handlers=com.google.cloud.logging.LoggingHandler
   ```
+
+  #### Alternative way to ingest logs in Google Cloud managed environments
+
+  If you use Java logger with the Cloud Logging Handler, you can configure the handler to output logs to `stdout` using
+  the [structured logging Json format](https://cloud.google.com/logging/docs/structured-logging#special-payload-fields).
+  To do this, add `com.google.cloud.logging.LoggingHandler.redirectToStdout=true` to the logger configuration file.
+  You can use this configuration when running applications in Google Cloud managed environments such as AppEngine, Cloud Run,
+  Cloud Function or GKE. The logger agent installed on these environments can capture STDOUT and ingest it into Cloud Logging.
+  The agent can parse structured logs printed to STDOUT and capture additional log metadata beside the log payload.
+  The parsed information includes severity, source location, user labels, http request and tracing information.
+
+  #### Auto-population of log entrys' metadata
+
+  LogEntry object metadata information such as [monitored resource](https://cloud.google.com/logging/docs/reference/v2/rest/v2/MonitoredResource), 
+  [Http request](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#HttpRequest) or 
+  [source location](https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#LogEntrySourceLocation)
+  are automatically populated with information that the library retrieves from the execution context.
+  The library populates only empty (set to `null`) LogEntry fields.
+  This behavior can be opted out by calling `LoggingOptions.Builder.setAutoPopulateMetadata(false)`.
+  Cloud Logging handler can be configured to opt-out automatic population of the metadata via the config file by
+  setting `com.google.cloud.logging.LoggingHandler.autoPopulateMetadata` configuration to `false`.
+
+  The auto-population logic populates source location _only_ for log entries with `Severity.DEBUG` severity.
+  The execution context of the Http request and tracing information is maintained by `ContextHandler` class.
+  The context is managed in the scope of the thread.
```

Comment:
Perhaps you should add also the explanation about context stored into thread local storage (with/without inheritance) for manual case. It would be great if you can add the configuration sample as well for `getUseInheritedContext`

Reply:
Will do :+1: